modules/installing-ocp-agent-verify.adoc

```diff
@@ -0,0 +1,90 @@
+// Module included in the following assemblies:
+//
+// * installing-with-agent/installing-with-agent.adoc
+
+:_content-type: PROCEDURE
+[id="installing-ocp-agent-verify_{context}"]
+= Tracking and verifying installation progress
+
+Use the following procedure to track installation progress and to verify a successful installation.
+
+.Procedure
+
+. Optional: To know when the bootstrap host (rendezvous host) reboots, run the following command:
+
++
+[source,terminal]
+----
+$ ./openshift-install --dir <install_directory> agent wait-for bootstrap-complete \ <1>
+    --log-level=info <2>
+----
+<1> For `<install_directory>`, specify the path to the directory where the agent ISO was generated.
+<2> To view different installation details, specify `warn`, `debug`, or `error` instead of `info`.
+
++
+.Example output
+[source,terminal]
+----
+...................................................................
+...................................................................
+INFO Bootstrap configMap status is complete
+INFO cluster bootstrap is complete
+----
++
+The command succeeds when the Kubernetes API server signals that it has been bootstrapped on the control plane machines.
+
+. To track the progress and verify successful installation, run the following command:
++
+[source,terminal]
+----
+$ openshift-install --dir <install_directory> agent wait-for install-complete <1>
+----
+<1> For `<install_directory>` directory, specify the path to the directory where the agent ISO was generated.
+
++
+.Example output
+[source,terminal]
+----
+...................................................................
+...................................................................
+INFO Cluster is installed
+INFO Install complete!
+INFO To access the cluster as the system:admin user when using 'oc', run
+INFO     export KUBECONFIG=/home/core/installer/auth/kubeconfig
+INFO Access the OpenShift web-console here: https://console-openshift-console.apps.sno-cluster.test.example.com
+----
+
+
+[NOTE]
```

Comment:
Does this note and the dual-stack example belong here? This is a verification step.  Both are present in a previous module.